package dto.exceptions;

public class IdAlreadyInUseException extends RuntimeException {
}